docs/conf/portland/2019/unconference.rst

```diff
@@ -6,11 +6,19 @@ Unconference
 
 {% include "conf/events/unconference.rst" %}
 
+Unconference sessions are held downstairs from the Crystal Ballroom where the main stage talks are held, in Lola's Room. Because the unconference has become so popular in recent years, we ask session organizers to bear the following guidelines in mind:
+
+* Keep your sessions manageably small. If attendance starts to spill into neighboring tables, consider asking folks to attend an additional session on the same topic.
+* If you're leading a session on a popular topic (notably API docs in the past), consider scheduling an early time so there's room to schedule overflow sessions as needed later in the day, or even the next day if the first session is on Monday.
+* Ask unconf organizers to help wrangle logistics. Lola's Room can get crowded especially when a popular topic is scheduled.
+
+
+Unconf organizers (link to us?) are also available to help at any time during the conference. Ask about a topic, ask about the schedule, ask about multiple instances of the same topic -- we want to help you make your unconf session the best it can be. If you can't find us in Lola's room, feel free to seek us out elsewhere. There'll also be a dedicated volunteer who can help.
 
```

Comment:
To make sure I'm understanding: this means there are 2 unconf organizers and 1 dedicated volunteer, right?

Reply:
yes, that's right (confirmed with @alexfornuto the volunteer organizer). Needs clearer wording?

Reply:
I'm not sure it's unclear, but I did wonder what is the difference between the organizer and the volunteer. If there's really no distinction (at least from the view of the session leaders/participants), maybe it would be clearer to say something like:

Unconf organizers and a dedicated volunteer area also available to help at any time during the conference.

...and then delete that last sentence?

I'm overthinking this a bit, but that's one of my hobbies :D